apps/katib/upstream/components/controller/trial-templates.yaml

```diff
@@ -33,7 +33,7 @@ data:
         spec:
           containers:
             - name: training-container
-              image: docker.io/kubeflowkatib/enas-cnn-cifar10-cpu:v0.16.0-rc.1
+              image: docker.io/kubeflowkatib/enas-cnn-cifar10-cpu:v0.16.0
```

Comment:
Looks like this image tag does not exist.

cc @andreyvelich

Reply:
I am just merging what we have in 1.8.

Reply:
Why this tag doesn't exist ?
Here it is: https://hub.docker.com/layers/kubeflowkatib/enas-cnn-cifar10-cpu/v0.16.0/images/sha256-5c27e2c36638d0764619d99c6de442ee54a8991c6bc5e0c8e5dacb9a75ddd9ed?context=repo

Reply:
Hmm, I tried search for this tag in Docker Hub but can't find it. Thanks for the confirmation!